requirements-base.txt

```diff
@@ -67,3 +67,5 @@ redis-py-cluster>=1.3.4,<1.4.0
 jsonschema==2.6.0
 # for bitbucket client
 unidiff>=0.5.4
+# for vsts repo
+mistune>0.7,<0.9
```

Comment:
@mattrobenolt We only use mistune in VSTS to handle markdown of a Vsts description. Looking at the change log, we don't seem to use any features that have been affected. I see no reason not to include the most recent version.

Reply:
Cool, thanks for looking into this. We should follow up after this is merged on our internal `getsentry` version which is pinned at `0.7.2` and get it to `0.8.3`.